frontend/src/modules/report/pages/templates/report-template-page.vue

```diff
@@ -137,7 +137,7 @@ const initialPlatformValue = {
 };
 
 const platform = ref(initialPlatformValue);
-const teamMembers = ref(false);
+const teamOrganizations = ref(false);
```

Comment:
Maybe this was a mistake? I think this will break the filters since in the same file we are still using `teamMembers`